controllers/base_controller.js

```diff
@@ -67,6 +71,17 @@ module.exports = function BaseControllerModule(pb) {
      * @type {String}
      */
     var ALERT_PATTERN = '<div class="alert %s error_success">%s<button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button></div>';
+
+    /**
+     * The prefix in the content-type header that indicates the charset used in 
+     * the encoding
+     * @static
+     * @private
+     * @readonly
+     * @property CHARSET_HEADER_PREFIX
+     * @type {String}
+     */
+    var CHARSET_HEADER_PREFIX = 'charset=';
```

Comment:
this variable isn't used anywhere in the file and is private.  it should be removed.